app/assets/javascripts/media_player_wrapper/mejs4_helper_markers.es6

```diff
@@ -358,27 +343,91 @@ class MEJSMarkersHelper {
     }
   }
 
+  /**
+   * Re-build the markers table and player markers
+   * @function rebuildMarkers
+   * @return {void}
+   */
+  rebuildMarkers() {
+    this.rebuildMarkersTable();
+    this.updateVisualMarkers();
+  }
+
   /**
    * Update markers in Mediaelement player's time rail by hooking into the Mediaelement Markers plugin
    * @function updateVisualMarkers
    * @param {Array} markers Array of marker start times
    * @return {void}
    */
-  updateVisualMarkers(markers) {
-    const t = this;
-    const player = mejs4AvalonPlayer.player;
-    player.options.markers = markers;
+  updateVisualMarkers() {
```

Comment:
If we're in a time crunch, it's fine but ideally we should break this function down some it seems.  Does it pass eslint?